build.sbt

```diff
@@ -15,11 +15,36 @@ ThisBuild / publishAsOSSProject := true
 lazy val publishTestsSettings = Seq(
   Test / packageBin / publishArtifact := true)
 
+lazy val quasarVersion =
+  Def.setting[String](managedVersions.value("precog-quasar"))
+
+val Specs2Version = "4.9.4"
+val SLF4SVersion = "1.7.25"
+
 lazy val root = project
   .in(file("."))
   .settings(noPublishSettings)
-  .aggregate(core)
+  .aggregate(datasource)
+
+lazy val datasource = project
+  .in(file("datasource"))
```

Comment:
I think all the projects now are storing source code in `core`. Is there reason we are deviation from that pattern in this project?

Reply:
No reason to deviate. I saw that the CI build file default file expected `core` as well, so I'll switch over to that.